plugin.py

```diff
@@ -18,21 +21,29 @@ class LspPyrightPlugin(NpmClientHandler):
     server_directory = "language-server"
     server_binary_path = os.path.join(server_directory, "node_modules", "pyright", "langserver.index.js")
 
-    @classmethod
-    def additional_variables(cls) -> Optional[Dict[str, str]]:
-        variables = super().additional_variables() or {}
-        variables.update(
-            {
-                "sublime_py_files_dir": os.path.dirname(sublime.__file__),
-            }
-        )
-
-        return variables
-
     @classmethod
     def install_in_cache(cls) -> bool:
         return False
 
     @classmethod
     def minimum_node_version(cls) -> Tuple[int, int, int]:
         return (12, 0, 0)
+
+    @classmethod
+    def on_client_configuration_ready(cls, configuration: Dict[str, Any]) -> None:
+        super().on_client_configuration_ready(configuration)
+
+        if get_setting("st_plugin_development_mode"):
+            configuration["settings"]["python.analysis.extraPaths"].extend(cls.find_package_dependency_dirs())
+
+    @staticmethod
+    def find_package_dependency_dirs() -> List[str]:
+        dep_dirs = sys.path.copy()
+
+        # move the "Packages/" to the last
+        # @see https://github.com/sublimelsp/LSP-pyright/pull/26#discussion_r520747708
+        packages_path = sublime.packages_path()
+        dep_dirs.remove(packages_path)
+        dep_dirs.append(packages_path)
```

Comment:
Should we also remove .zip paths, paths from `Installed Packages`, and paths from `.sublime-package` files? Or maybe pyright simply understands those paths :) I don't know.

Reply:
Since `.sublime-package` never works, I think it's good to remove them!

Reply:
Resolved in https://github.com/sublimelsp/LSP-pyright/pull/26/commits/bc03321b030b5700759fec572807667b7a451cd5